### PR TITLE
VRVU-28: add nutrition zscore

### PR DESCRIPTION
**Pull request by jdolkowski**
https://sd-vxnaid.atlassian.net/browse/VRVU-28

### CODE REVIEW

File: app/src/main/java/com/jnj/vaccinetracker/visit/zscore/NutritionZScoreCalculator.kt [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+      if (zScore <= -3) return true
+      return false
+   }
+
+   fun getTextColorBasedOnZsCoreValue(): Int {
+      val defaultColor = R.color.colorTextOnLight
+      val zScore = this.zScore ?: return defaultColor
+      val color = when {
+         zScore <= -3 -> Color.RED
+         zScore < -2.0 -> Color.YELLOW
```
Comment (sherrif10) [inline app/src/main/java/com/jnj/vaccinetracker/visit/zscore/NutritionZScoreCalculator.kt]: Lets change this to something **black** or different color ,   Yellow isnt good.  Check out 
Screenshot 2024-06-13 070048

Comment (jdolkowski) [reply app/src/main/java/com/jnj/vaccinetracker/visit/zscore/NutritionZScoreCalculator.kt]: that was a requirement, however i can change it to a darker shade

Comment (sherrif10) [reply app/src/main/java/com/jnj/vaccinetracker/visit/zscore/NutritionZScoreCalculator.kt]: Its not readable as you can see. Thanks

**Comment by jdolkowski** [comment]
Now it should be visible.

**Pull request merged**